pkg/bff/config/config.go

```diff
@@ -76,8 +86,8 @@ type DatabaseConfig struct {
 }
 
 type MTLSConfig struct {
-	CertPath string `split_words:"true" required:"true"`
-	PoolPath string `split_words:"true" required:"true"`
+	CertPath string `split_words:"true"`
+	PoolPath string `split_words:"true"`
```

Comment:
Maybe we should move the `Insecure` bool here since the certs aren't required anymore?

Reply:
Sure, I can do that.